### DISCUSSION TITLE
How to convert an ase object or pymatgen object into Geometry?

### DISCUSSION THREAD

**Discussion by PramSin** [zerothi/sisl#554] [Q&A] [answered]
In you documentation, I see I can convert a Geometry() to an ase.Atoms() by
```python
g = sisl.geom.graphene()
g.to.ase()
``` 
But I don't know how to do the inverse process. The release notes 0.12.0 said it complemented from/to pymatgen. But I still don't know how to use the "from" method. Looking forward to your answers! Thanks for your fantastic codes btw.

**Comment by PramSin** [comment, marked as answer]
After reading the source code, I found the method should be `sisl.Geometry.new.pymatgen()`. But I still failed to complete the convertion. I will create an issue about the details later. Now I have to convert it into ase and then to pymatgen.

**Reply by zerothi** [reply]
Thanks, I will try out to see why this happens, it would really help if you could give me the error output for figuring out why it happens.

Thanks for your feedback! :)

**Reply by zerothi** [reply]
I have checked, indeed there was an error. It should be fixed in the latest commit. Thanks!
